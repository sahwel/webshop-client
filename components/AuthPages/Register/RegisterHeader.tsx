import React from 'react';

const RegisterHeader = () => {
  return (
    <div className="select-none">
      <h1 className="text-[300%]">Let’s sign up!</h1>
      <h3>We are happy you’re</h3>
      <h3>joining us!</h3>
    </div>
  );
};

export default RegisterHeader;
