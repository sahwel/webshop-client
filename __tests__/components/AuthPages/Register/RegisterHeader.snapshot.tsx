import { render, renderHook, RenderHookResult } from '@testing-library/react';
import RegisterHeader from '@/components/AuthPages/Register/RegisterHeader';

describe('RegisterHeader Snapshot', () => {
  it('renders RegisterHeader unchanged', () => {
    const { container } = render(<RegisterHeader />);
    expect(container).toMatchSnapshot();
  });
});
