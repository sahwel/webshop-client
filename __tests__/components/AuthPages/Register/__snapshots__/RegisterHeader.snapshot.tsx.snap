// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterHeader Snapshot renders RegisterHeader unchanged 1`] = `
<div>
  <div
    class="select-none"
  >
    <h1
      class="text-[300%]"
    >
      Let’s sign up!
    </h1>
    <h3>
      We are happy you’re
    </h3>
    <h3>
      joining us!
    </h3>
  </div>
</div>
`;
